### DISCUSSION TITLE
Primer3: tool to select seq between primers

### DISCUSSION THREAD

**Discussion by blex-max** [blex-max/ecto_pipe#17] [Ideas]
https://github.com/primer3-org/primer3

This tool is very interesting to me but my lack of molecular biology knowledge means I can't really assess it's usefulness - basically, given a set of primer pairs and a sequence it will identify those pairs in the sequence and extract the region of interest. Does that sound like a good way to discard the garbage we get at the start/end of sequences?

https://www.gear-genomics.com/primer3plus/ - here's a web app version of it so you can see how it works.

@catnipping @lauramsuz @clarkson1976

**Comment by catnipping** [comment]
Primer3 is a tool widely used to design and test PCR primers _in silico_. (or at least, it was used when I was studying) and to test for possible misamplification (i.e. amplifying the wrong region due to low specificity). 
I don't see it as a good tool to pick primer regions in our sequences; usually, primers fall into the poor quality regions, and it's difficult to find and exact match just by searching the primer sequence. The program won't find any match unless the sequence matches perfectly. I think primers are being dropped just by trimming poor quality regions using the phred scores. Does it make sense?
